### PR TITLE
Add Table of Content to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Tranberry**
A simple ToS with links to the different parts of the CONTRIBUTING.md just like in README.md

### CODE REVIEW

**Comment by zwrawr** [comment]
looks good

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+[Contributing](#contributing)  
+[Resolving Merge Conflicts](#resolving-merge-conflicts)
+[Unity Version](#unity-version)
```
Comment (Tranberry) [inline CONTRIBUTING.md]: needs 'new line' as "unity version" gets placed on the same line as "resolving merge.."

**Comment by RodRitter** [comment]
If you fix the new line issue, I'll merge this in.

**Comment by Tranberry** [comment]
@RodRitter should I ping or not after fixing? It should be fixed now.

**Comment by RodRitter** [comment]
It's cool. Merging.

**Pull request merged**